.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: csharp
+solution: MoreLinq.sln
+mono: latest
+dist: trusty
+sudo: required
+dotnet: 1.0.0-preview2-003131
+
+install:
+  - dotnet restore
+
+script:
+  - dotnet build -c Debug MoreLinq
```

Comment:
....And yes, it appears it was a typo :speak_no_evil: . Now we build and test debug and release configurations.